src/server/models/Meter.js

```diff
@@ -15,14 +15,16 @@ class Meter {
 	 * @param enabled This meter is being actively read from
 	 * @param displayable This meters is available to users for charting
 	 * @param type What kind of meter this is
+	 * @param meterTimezonep Default timezone for meter
```

Comment:
imezonep typo

Reply:
I fixed this while merging in development. It will be seen once I finish and push out my changes.